lib/rubygems/install_update_options.rb

```diff
@@ -158,10 +158,9 @@ def add_install_update_options
       options[:without_groups].concat v.map(&:intern)
     end
 
-    add_option(:"Install/Update", "--default",
+    add_option(:Deprecated, "--default",
                "Add the gem's full specification to",
                "specifications/default and extract only its bin") do |v,_o|
-      options[:install_as_default] = v
     end
```

Comment:
Should we use `deprecate_option` here, to also give a runtime deprecation message? The way it is right now, I think it only appears as deprecated in documentation.